README.md

```diff
@@ -31,7 +41,7 @@ Flight icons in different formats (SVG/SVG Sprite/React)
 Design tokens
 
 - npm package: `@hashicorp/design-system-tokens`
-- website: [design-system-components-hashicorp.vercel.app/foundations/tokens](https://design-system-components-hashicorp.vercel.app/foundations/tokens)
+- website: [https://helios.hashicorp.design/foundations/tokens](https://helios.hashicorp.design/foundations/tokens)
 - more info: see [tokens/README](packages/tokens/README.md) and [tokens/CONTRIBUTING](packages/tokens/CONTRIBUTING.md) for details on how to update the design tokens.
 
 ### `flight-website`
```

Comment:
Open to the idea of also just removing this section at this point as this site won't exist much longer